### DISCUSSION TITLE
Rainbow Kit Modal affects application CSS

### DISCUSSION THREAD

**Discussion by korlaism** [rainbow-me/rainbowkit#1463] [Q&A]
I have the rainbowkit css imported at the top of application and am using the default lightTheme with the provider to configure. I have my own button which uses the `openConnectModal` hook to show the modal. Note that I have tried version 1.0.4 to 1.0.8 and have the same issue with all of them. Now the issue is that when the hook is called to display the modal, functionally everything works, including the connection, but my application fonts are affected, this includes font-family, color and font-weight. It remains so even once the account is connected, but is reset to my custom styling once I refresh the page. I wanted to have a discussion before I created an issue. Any help or recommendation will be greatly appreciated. Thanks

**Comment by DanielSinclair** [comment]
@korlaism Leaking styling like that shouldn't be the case. If possible, please open an issue with a minimal CodeSandbox example that shows the problem, and we'll investigate!
